.circleci/config.yml

```diff
@@ -43,8 +43,8 @@ jobs:
             # Holiday dates specific to 2022 that will need to be updated next year
             # as per https://intranet.broadinstitute.org/keywords/holiday-calendar
             # and https://intranet.broadinstitute.org/hr/human-resources-policy-manual/time
-            HOLIDAYS=("12/23/2021" "01/17/2022" "02/21/2022" "04/18/2022" "05/30/2022" "07/04/2022"
-            "09/05/2022" "10/10/2022" "11/11/2022" "11/24/2022" "11/25/2022" "12/23/2022")
+            HOLIDAYS=("12/23/2021" "01/17/2022" "02/21/2022" "04/18/2022" "05/30/2022" "07/01/2022" "07/04/2022"
+            "09/02/2022" "09/05/2022" "10/10/2022" "11/11/2022" "11/24/2022" "11/25/2022" "12/23/2022")
```

Comment:
It seems like we're also missing `06/19/2022` (Juneteenth).

Reply:
Good catch. Added June 19th to the recurring holidays and June 20th to this year's holidays.